### ISSUE TITLE
Bug - Not using Proxy Setting at the port scanner naabu

### ISSUE THREAD

**Issue by Code-L0V3R** [yogeshojha/rengine#698]
### Issue Summary

Not using proxy setting at the port scanner naabu command when the user configured proxy setting from the web ui.

https://github.com/yogeshojha/rengine/blob/f31e7060dbc2e39442ee6b5f7648fb14137910f6/web/reNgine/tasks.py#L1053-L1057

celery logs
```
rengine-celery-1       | [2022-09-01 17:19:38,267: INFO/ForkPoolWorker-5] naabu -list /usr/src/scan_results/example.net_620608300667/sorted_subdomain_collection.txt -json -o /usr/src/scan_results/example.net_620608300667/ports.json -exclude-cdn  -top-ports 1000  -rate 1000 
```

* I have confirmed that this issue can be reproduced as described on a latest version/pull of reNgine: yes

**Comment by Code-L0V3R** [comment]
I made a PR
Please Merge https://github.com/yogeshojha/rengine/pull/697

**Comment by ocervell** [comment]
@yogeshojha this can be closed as the PR is merged.

**Comment by psyray** [comment]
Fixed by [2.0 version](https://github.com/yogeshojha/rengine/pull/963) with @ocervell PR https://github.com/yogeshojha/rengine/pull/742
https://rengine.wiki/changelog/#200
